build-system/babel-plugins/babel-plugin-transform-version/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const {VERSION} = require('../../internal-version') ;
+
+module.exports = function(babel) {
+  const {types: t} = babel;
+  return {
+    name: 'transform-version',
+    visitor: {
+    CallExpression(path) {
+      if (path.node.callee.name === 'version') {
```

Comment:
Maybe we should rename this function to `internalRuntimeVersion()`?

https://github.com/ampproject/amphtml/blob/9d6477b4b638f96c52453d7c19492b693e561bdb/src/internal-version.js#L26-L28

That will make the transform less likely to inadvertently replace the wrong function. There are 26 call sites in 14 files, so it shouldn't be too hard to replace them all.